sdcm/stress_thread.py

```diff
@@ -167,8 +167,6 @@ def _run_stress(self, node, loader_idx, cpu_idx, keyspace_idx):
                 #  pylint: disable=no-member
                 if hasattr(exc, 'result') and exc.result.failed:
                     stderr = exc.result.stderr
-                    if len(stderr) > 100:
```

Comment:
what's this change?

Reply:
Temporary fix to see stress_cmd stderr, reverted